src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SliderRows/SliderPanel/SliderPanel.types.ts

```diff
@@ -0,0 +1,7 @@
+import { FieldValues, UseControllerProps } from 'react-hook-form';
+
+export type SliderPanelProps<T extends FieldValues> = {
+  label: string;
+  isMultiple: boolean | undefined;
```

Comment:
could you pls try to use just isMultiple?: boolean ?

Reply:
done